crypto/evp/evp_fetch.c

```diff
@@ -504,6 +504,9 @@ char *evp_get_global_properties_str(OSSL_LIB_CTX *libctx, int loadconfig)
     char *propstr = NULL;
     size_t sz;
 
+    if (plp == NULL)
+        return "";
```

Comment:
This would have to be `return OPENSSL_strdup("");`

Reply:
Done.